charts/mattermost-team-edition/templates/service.yaml

```diff
@@ -1,6 +1,10 @@
 apiVersion: v1
 kind: Service
 metadata:
+{{- if .Values.service.annotations }}
+  annotations:
+{{ toYaml .Values.service.annotations | indent 4 }}
```

Comment:
can you add this to the values.yaml and document?

Reply:
done